documentation/source/documentation/cheatsheets/python.rst

```diff
@@ -0,0 +1,600 @@
+|                                  |      body                                |
+|                                  |    end                                   |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    x, y = exp                    |    let (x, y) = exp                      |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    x, y = (5, 6)                 |    let (x, y) = values(5, 6);            |
+|    f(x, y)                       |    f(x, y)                               |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    def foo():                    |    define function foo ()                |
+|      def f(x):                   |      local method f (x)                  |
+|        f-body                    |              f-body                      |
+|                                  |            end,                          |
+|      def g(y, z):                |            method g (y, z)               |
+|        g-body                    |              g-body                      |
+|                                  |            end;                          |
+|      body                        |      body                                |
+|                                  |    end;                                  |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    if test:                      |    if (test)                             |
+|      then1                       |      then1;                              |
+|      then2                       |      then2;                              |
+|    else:                         |    else                                  |
+|      else1                       |      else1;                              |
+|      else2                       |      else2;                              |
+|                                  |    end if                                |
++----------------------------------+------------------------------------------+
+| .. code-block:: python           | .. code-block:: dylan                    |
+|                                  |                                          |
+|    var = value                   |    var := value                          |
```

Comment:
How about removing this row and including both `var := value` and `let var = value` in the first row of this table with `define constant` et al?

Python really overloads the hell out of the `=` operator.